scratch/benchmarks/benchmark_fgpu.py

```diff
@@ -172,7 +168,7 @@ def fgpu_factory(
         f"--prometheus-port={prometheus_port} "
         f"--sync-time={sync_time} "
         f"--feng-id={index} "
-        f"{'--use-vkgdr --max-delay-diff=65536' if args.use_vkgdr else ''} "
+        f"{'--use-vkgdr' if args.use_vkgdr else ''} "
```

Comment:
You don't need `--max-delay-diff` anymore?

Reply:
That was to save memory.